app/ui.js

```diff
@@ -608,15 +608,64 @@ var UI;
             if (anchor.classList.contains("noVNC_right")) {
                 WebUtil.writeSetting('controlbar_pos', 'left');
                 anchor.classList.remove("noVNC_right");
+
+                UI.updateControlBarHint('right');
             } else {
                 WebUtil.writeSetting('controlbar_pos', 'right');
                 anchor.classList.add("noVNC_right");
+
+                UI.updateControlBarHint('left');
             }
 
             // Consider this a movement of the handle
             UI.controlbarDrag = true;
         },
 
+        displayControlBarHint: function (show) {
+            var handleHint = WebUtil.getSilhouette("noVNC_control_bar_handle");
+            var controlBarHint = WebUtil.getSilhouette("noVNC_control_bar");
+
+            if (!show) {
+                if (handleHint === null || controlBarHint === null) return;
+
+                handleHint.style.display = "none";
+                controlBarHint.style.display = "none";
```

Comment:
We did a lot of work to get stuff like this out of the code an in to CSS. It would be a shame if we regress on that.

Any reason why you cannot put the look of the hint in CSS and just let JavaScript control when it is visible?